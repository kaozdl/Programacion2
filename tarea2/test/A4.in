insertar_lista (100,dragon) (50,perro) (30,gallo) .
ordenar_lista
recorrido
liberar_lista
Fin
