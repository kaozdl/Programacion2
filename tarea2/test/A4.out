1>(0,primero)(100,dragon)(50,perro)(30,gallo)
2>4
(0,primero)(100,dragon)(50,perro)(30,gallo)
(0,primero)(30,gallo)(50,perro)(100,dragon)
(0,primero)(30,gallo)(50,perro)(100,dragon)
3>(0,primero)(100,dragon)(50,perro)(30,gallo)(30,gallo)(50,perro)(100,dragon)(0,primero)
4>5>Fin.
